### PR TITLE
aruha-333: changes to stats endpoint

### PR DESCRIPTION
**Pull request by adyach**

### CODE REVIEW

**Comment by rcillo** [comment]
:shipit:

**Comment by adyach** [comment]
@v-stepanov @rcillo @v1ctor added fixes for comments in merged pr: https://github.com/zalando/nakadi/pull/402/files

File: src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java [hunk 1]
```diff
@@ -214,42 +215,63 @@ private Callback kafkaSendCallback(final BatchItem item, final CountDownLatch do
...
-            final Map<Integer, Long> earliestOffsets = getPositions(consumer, kafkaTPs);
+            return toNakadiTopicPartition(consumer, kafkaTPs);
+        } catch (final Exception e) {
+            throw new ServiceUnavailableException("Error occurred when fetching partitions offsets", e);
+        }
+    }
 
-            consumer.seekToEnd(tpArray);
-            final Map<Integer, Long> latestOffsets = getPositions(consumer, kafkaTPs);
+    private List<TopicPartition> toNakadiTopicPartition(final Consumer<String, String> consumer,
```
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java]: I think the name of method doesn't fully reflect what happens.
I think the better name can be something like `getPartitionsBounds` or something like that

**Comment by v-stepanov** [comment]
:shipit:

**Comment by adyach** [comment]
@v-stepanov @rcillo merged with master, reapprove, please,

**Comment by rcillo** [comment]
👍

**Pull request merged**